website/src/components/misc/global-style.tsx

```diff
@@ -35,12 +35,10 @@ export const GlobalStyle = createGlobalStyle`
 
     > div:first-child {
       display: block;
-      height: 100vh;
```

Comment:
removing height leads to another issue, where navigation does not take anymore the entire height of the content area.